cicd/jenkins-basic/docker/Dockerfile

```diff
@@ -18,17 +18,16 @@ ARG TINI_VERSION=v0.18.0
 ARG TINI_QUALIFIER=amd64
 ARG SWARM_CLIENT_VERSION=3.15
```

Comment:
We probably need to update the jenkins swarm plugin as well. The latest is `3.23`. If you don't want to do that in this PR, maybe create an issue for later.